uint32 type
  uint32 FILTER_NONE         = 0
  uint32 FILTER_BOUNDING_BOX = 1
  uint32 FILTER_SPHERE       = 2

float32[3] bbox_min
float32[3] bbox_max
float32[3] center
float32    radius
